tasks/sgx.py

```diff
@@ -15,7 +15,10 @@ def do_configure(repo_url, write_file, key_url):
 
 
 @task
-def configure():
+def configure(ctx):
```

Comment:
This file was comitted before, feel free to leave comments on any line, even if it is not part of this `diff`.